railgun/hack/generators/controllers/networking/main.go

```diff
@@ -280,7 +280,7 @@ func (needed necessary) generate() error {
 		}
 	}
 
-	return ioutil.WriteFile(outputFile, contents.Bytes(), 0644)
+	return ioutil.WriteFile(outputFile, contents.Bytes(), 0600)
```

Comment:
0600 means this file cant get updated/executed later on, permission change. Are you sure ?

Reply:
This file can be updated, but only by the user. The file was never executable. Yes, `0600` should be fine, it was what was suggested by `golangci-lint`.

Reply:
Talked with @ccfishk in Slack, we're all set